lib/sitemap/generator.rb

```diff
@@ -83,7 +83,23 @@ def get_all_documents
         documents.map do |document|
           chunk << SitemapPresenter.new(document["_source"], property_boost_calculator)
 
-          if chunk.size == SITEMAP_LIMIT
+          parts = document["_source"]["parts"] || []
+
+          parts.each do |part|
+            part_document = document["_source"].merge(
+              "is_part" => true,
+              "link" => document["_source"]["link"] + "/" + part["slug"],
+            )
+
+            chunk << SitemapPresenter.new(part_document, property_boost_calculator)
+
+            if chunk.size >= SITEMAP_LIMIT
```

Comment:
Not sure I follow why it was `if chunk.size == SITEMAP_LIMIT` before but `>=` now

Reply:
Ah yes, I think I was worried at one point that I might miscount somehow and potentially end up exceeding the limit (by adding more than one part before checking the chunk size for example). 
It felt safer (for this, and future changes) to make the check do `>=` so that it would be able to work even if I'd cocked up the count somewhere.